tutorials/io/runtime_file_loading_and_saving.rst

```diff
@@ -168,8 +168,8 @@ Example of loading an image and displaying it in a :ref:`class_TextureRect` node
 Audio/video files
 -----------------
 
-Godot supports loading Ogg Vorbis audio at runtime. Note that not *all* files
-with an ``.ogg`` extension may be Ogg Vorbis files. Some may be Ogg Theora
+Godot supports loading Ogg Vorbis and WAV audio at runtime. Note that not *all*
+files with an ``.ogg`` extension may be Ogg Vorbis files. Some may be Ogg Theora
```

Comment:
```suggestion
files with an ``.ogg`` extension are Ogg Vorbis files. Some may be Ogg Theora
```

Reply:
AFAIK, OGG is the container, and *may* contain Ogg Vorbis encoded audio, or Ogg Theora encoded video, or both? So this is technically correct (leaving aside that technically, *any* file with *any* extension may contain anything at all :))

Reply:
It's more about the wording than the correctness of the original, "may" means "is permitted", the use of the word to mean "is possible" is non-standard and potentially misleading (been meaning to add it to the list of rules for documentation as it's not recommended generally)